stack/Cpp/Test.cpp

```diff
@@ -0,0 +1,18 @@
+#include<iostream>
+#include "Stack.cpp"
+using namespace std;
+
+int main() {
+	Stack<int> s;
+	s.push(1);
+	s.push(2);
+	s.push(3);
+	s.pop();
+	cout << s.top() << endl;
+	cout << s.size()<<endl;
+	s.clear();
+	cout << s.size()<<endl;
+	s.push(15);
+	cout << s.top() << endl;
+	return 0;
+}
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpbn9qh2vv/stack/Cpp/Test.cpp
+++ b/tmp/tmpbn9qh2vv/stack/Cpp/Test.cpp
@@ -15,4 +15,4 @@
 	s.push(15);
 	cout << s.top() << endl;
 	return 0;
-}+}
```